### ISSUE TITLE
Add option to directly reply the reply of a single host

### ISSUE THREAD

**Issue by nritsche** [chime-experiment/coco#217]
For https://github.com/chime-experiment/Infrastructure/issues/154 coco needs a report option to simplify a reply from a single host to just that reply, without the typical coco report overhead:

```
["cn3g2:12048", "cn3g8:12048", "cs8g0:12048"]
```
instead of
```
{"success": true, "blocklist": {"http://coco/": {"reply": ["cn3g2:12048", "cn3g8:12048", "cs8g0:12048"], "status": 200}}}
```

**Comment by andrerenard** [comment]
Can we also remove the port number on the host?

**Comment by nritsche** [comment]
> Can we also remove the port number on the host?

Yes, but that should probably a completely separate option: https://github.com/chime-experiment/coco/issues/218
